src/Shared/PlatformSupport/AssetLoader.cs

```diff
@@ -83,7 +83,7 @@ public bool Exists(Uri uri, Uri baseUri = null)
         /// <exception cref="FileNotFoundException">
         /// The resource was not found.
         /// </exception>
-        public Tuple<Stream, Assembly> OpenAndGetAssembly(Uri uri, Uri baseUri = null)
+        public (Stream Stream, Assembly Assembly) OpenAndGetAssembly(Uri uri, Uri baseUri = null)
```

Comment:
Nit:
Could we use `camelCase` for tuple values. It would look odd when a tuple is deconstructed and you end up with locals that start with capitals.

Reply:
I will change that.